src/mate_clj/core.clj

```diff
@@ -156,3 +156,50 @@
   ([pred] (dfilter (complement pred)))
   ([pred coll]
    (dfilter (complement pred) coll)))
+
+(defn dtake-while
+  ([pred]
+     (fn [rf]
+       (fn
+         ([] (rf))
+         ([result] (rf result))
+         ([result input]
+            (println pred input " => " (pred input))
```

Comment:
remove spaces from `" => "` to be `"=>"`

Reply:
done